modules/FvwmMFL/FvwmMFL.c

```diff
@@ -246,7 +163,7 @@ static void
 HandleTerminate(int fd, short what, void *arg)
 {
 	fprintf(stderr, "%s: dying...\n", __func__);
-	delete_pid_file();
```

Comment:
Probably close `fvwmmfl_lock_fd` here? And if only to show that you have not forgotten :-).  Also in the other place(s) where you unlink.